gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CredentialsController.groovy

```diff
@@ -41,14 +43,18 @@ class CredentialsController {
   @PostFilter("hasPermission(filterObject.name, 'ACCOUNT', 'READ')")
   @ApiOperation(value = "Retrieve a list of accounts")
   @RequestMapping(method = RequestMethod.GET)
-  List<ClouddriverService.Account> getAccounts(@SpinnakerUser User user) {
-    credentialsService.getAccounts(user?.roles ?: [])
+  List<Account> getAccounts(@SpinnakerUser User user, @RequestParam(value = "expand", required = false) boolean expand) {
+    List<AccountDetails> allAccounts = credentialsService.getAccounts(user?.roles ?: [])
+    if (expand) {
+      return allAccounts
+    }
+    return allAccounts.collect { Account.from(it) }
```

Comment:
The only reason I'm doing this is to avoid changing what's currently returned from calling `/credentials`. I tried using `objectMapper.convertValue(it, Account)`, but Jackson left the class alone.

EDIT: I've updated this to:
```groovy
return objectMapper.convertValue(allAccounts, new TypeReference<List<Account>>() {})
```
and removed the `.from` method from the Account class. I think it reads a little better and still preserves the existing shape of the response.